### DISCUSSION TITLE
part-ii/2018-21/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#260] [Question comments]
Part II 2018 Asymptotic Methods: Paper 4, Section II, B 

https://questions.tripos.org/part-ii/2018-21/

**Comment by acornnroca** [comment]
<details><summary>Asymptotic expansion of I_0</summary>

$$\frac{e^x}{2} \sqrt{\frac{2}{\pi x} + \frac{x^{-3/2}e^x}{8\sqrt{2 \pi}}$$


</details>
